lib/sqlalchemy/dialects/sqlite/base.py

```diff
@@ -1032,6 +1032,9 @@ def visit_empty_set_expr(self, element_types):
 
 class SQLiteDDLCompiler(compiler.DDLCompiler):
     def get_column_specification(self, column, **kwargs):
+        if column.computed is not None:
```

Comment:
**mike bayer** (zzzeek) wrote:

maybe just let it get passed through to SQLite and let it raise?   because SQLite might add this feature.

Reply:
I raise the error so it is consistent with the other implementations that raise an error in case of an unsupported option. Like in oracle it raises an error if persisted is true.
If we do not raise an error here I think we also should not raise an error in case of an unsupported option in the other dialects

Reply:
**mike bayer** (zzzeek) wrote:

well when we see an option for which we don't know what SQL to generate, that's harder to ignore because we can't have it emit an error from the DB.   it's fine to leave this as is however.

Reply:
Ok